repository-data/application/src/main/resources/hcm-config/namespaces/visitscotland/ComparatorSystem.yaml

```
@@ -0,0 +1,158 @@
+      jcr:uuid: c243db6b-dfb4-402a-ac08-8be9e33317ce
+      /hipposysedit:nodetype:
+        jcr:primaryType: hippo:handle
+        jcr:mixinTypes: ['mix:referenceable']
+        jcr:uuid: 2fd36b82-66b8-4e10-9c94-52bddd5f214c
+        /hipposysedit:nodetype:
+          jcr:primaryType: hipposysedit:nodetype
+          jcr:mixinTypes: ['mix:referenceable', 'hipposysedit:remodel']
+          jcr:uuid: 980c11b4-c7eb-48b9-a6f7-c47718c7d968
+          hipposysedit:node: true
+          hipposysedit:supertype: ['visitscotland:basedocument', 'hippostd:relaxed',
+            'hippotranslation:translated']
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 35 but found 12

(indentation)